src/App.jsx

```diff
@@ -15,14 +22,24 @@ function App() {
   };
 
   return (
-    <ChakraProvider>
-      <DropZone setFiles={setFiles} />
-      <Button onClick={onSubmit}>Upload File</Button>
-      {images.map(e => (
-        <Image key={e} src={e} />
-      ))}
-      <DonationForm />
-    </ChakraProvider>
+    <Router>
+      <ChakraProvider>
+        <DropZone setFiles={setFiles} />
+        <Button onClick={onSubmit}>Upload File</Button>
+        {images.map(e => (
+          <Image key={e} src={e} />
+        ))}
+        <DonationForm />
```

Comment:
You can get rid of this since you just created a route that leads to the donation form already